docs/quickstart/data-subjects-api.md

```diff
@@ -0,0 +1,8 @@
+---
+title: Data Subjects API
+hide_table_of_contents: false
+sidebar_position: 4
+---
+
+A [recent blog-post](https://strmprivacy.io/posts/batchjobs-and-datasubjects/#dss) walks through a hands-on session with the Data
```

Comment:
Is the plan to extend this section later on?

Reply:
Maybe; not any time soon I think.